packages/hop-node/src/watchers/xDomainMessageRelayWatcher.ts

```diff
@@ -19,12 +23,20 @@ type Config = {
   dryMode?: boolean
 }
 
+type ConfirmRootData = {
+  rootHash: string
+  destinationChainId: number
+  totalAmount: string | BigNumber
+  rootCommittedAt: number
+}
+
 type Watcher = GnosisBridgeWatcher | PolygonBridgeWatcher | OptimismBridgeWatcher | ArbitrumBridgeWatcher
 
 class xDomainMessageRelayWatcher extends BaseWatcher {
   l1Bridge: L1Bridge
   lastSeen: {[key: string]: number} = {}
   watchers: {[chain: string]: Watcher} = {}
+  messengerWrapper: MessengerWrapper
```

Comment:
general suggestion; would prefer `messengerWrapper` to be labeled as `l1MessengerWrapper` where ever it's referenced in hop node to follow existing naming convention we have in core package

Reply:
I see now what you mean.
For consistency, and clarity, I do still prefer the `l1`/`l2` prefix and the other 2 instances you pointed out should have had the prefix as well. 

We currently use `L2Amm` and `L2AmmWrapper` for the classes which I like and the label `saddleSwap` can probably be generalized to `l2Amm` across the packages too but probably not worth the trouble changing it anyway